testing/nicotine-plus/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: dai9ah <dai9ah@protonmail.com>
+# Maintainer:
+pkgname=nicotine+
+_pkgname=nicotine-plus
+pkgver=1.4.1
+pkgrel=0
+pkgdesc="a graphical client for the SoulSeek peer-to-peer system"
+url="http://nicotine-plus.org/"
+arch="noarch"
+license="GPL-3.0"
+depends="librsvg py-gtk py-mutagen"
+subpackages="$pkgname-doc $pkgname-lang"
+source="$pkgname-$pkgver.tar.gz::https://github.com/Nicotine-Plus/$_pkgname/archive/$pkgver.tar.gz"
+
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
```

Comment:
no python3 support? I want avoid adding more python2 stuff if possible.

Reply:
Unfortunately their project currently seems to be python2. Since python2 is on the way out, i agree that we shouldn't merge.